src/mon/Monitor.cc

```diff
@@ -4304,7 +4304,6 @@ void Monitor::_ms_dispatch(Message *m)
         // exist only while the op is being handled.
         remove_session(s);
       }
-      s->put();
```

Comment:
i think we still need this "put()". "s" is retrieved from "op" using `op->get_session()` where we get a strong reference by calling `session.get()` in `op->get_session()`. the "set_session" call  in the next block is actually referencing a new session created by `session_map.new_session()`.

Reply:
```
  MonSession *get_session() const {
    return static_cast<MonSession*>(session.get());
  }
```
that session is
```
  RefCountedPtr session;
```
so IIUC the .get() is on the smart pointer object (returning the bare pointer), not the object (to adjust the ref count).